tests/unit/stream_alert_rule_processor/test_alert_forward.py

```diff
@@ -0,0 +1,209 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+# pylint: disable=protected-access
+from datetime import datetime
+import os
+
+from botocore.exceptions import ClientError
+from mock import ANY, call, patch
+from nose.tools import assert_equal
+
+from stream_alert.rule_processor.alert_forward import AlertForwarder
+from stream_alert.rule_processor.config import load_env
+from tests.unit.stream_alert_rule_processor.test_helpers import get_mock_context
+
+
+@patch.object(AlertForwarder, 'BACKOFF_MAX_RETRIES', 1)
+@patch.dict(os.environ, {'CLUSTER': 'corp'})
+class TestAlertForwarder(object):
+    """Test class for AlertForwarder"""
+    ALERT_PROCESSOR = 'corp-prefix_streamalert_alert_processor'
+    ALERT_TABLE = 'corp-prefix_streamalert_alerts'
+
+    @classmethod
+    def setup_class(cls):
+        """Setup the class before any methods"""
+        patcher = patch('stream_alert.rule_processor.alert_forward.boto3.client')
```

Comment:
not really an issue, but you don't have to use the fully-qualified path to the imported module here. `boto3.client` is sufficient

Reply:
This was just copied from the other file, but I can fix that